lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -59,29 +66,41 @@ def supported_features(cls) -> List[Type[Feature]]:
     def _initialize(self, *args: Any, **kwargs: Any) -> None:
         libvirt_events_thread.init()
 
+        self.qemu_platform_runbook = self.runbook.get_extended_runbook(
+            QemuPlatformSchema, type_name=QEMU
+        )
+        self.libvirt_conn_str = "qemu:///system"
+
+        if self.qemu_platform_runbook.is_host_remote():
+            self.host_node = RemoteNode(schema.Node(name="qemu-host"), 0, "qemu-host")
```

Comment:
You are welcome to also set the `local_node` as an object instance variable if it would help simplify the code.

Reply:
Actually, it looks `local_node` isn't even being used anymore.